### DISCUSSION TITLE
July 23 - Literate Programming V - your brand

### DISCUSSION THREAD

**Discussion by tonyfast** [Quansight/writers-workshop#25] [New class]
# the goal of the workshop

[you, your media, and your brand](https://gist.github.com/tonyfast/0c0b11c8e69bc8d38611ec6ac1fddbd2)

> this week are trying out jitsi. https://meet.jit.si/litprog-writers-workshopbounce-house

## so far

1. introduction to literate programs where code and narrative contribute equally to the narrative.
2. introduction to different media formats in the notebook and their use as collage.
3. introduction to testing a mess of notebooks.
4. restart and run all or it didn't happen

## teacher's preferred ground rules

* How to use the raise handle feature? Keeping the professor on track..
* I encourage y'all to interrupt me when I'm talking or demo'ing. 
  Sometimes I get distracted by my own special interests and this class if for you all.
* To help take notes, please respond to this discussion item and continue editing your one reply during class. i can see the changes on the discussion board as they happen.

## [calls to action] :+1: :-1:  :smile: :confused:  :rocket:  :heart:  :eyes: :tada: attendance

* place any questions you have in the discussions and we can try to discuss them during the workshop.

* react to this discussion so we can take attendance.

![image](https://user-images.githubusercontent.com/4236275/117461813-fd163180-af1b-11eb-901b-bfdee65d8bdc.png)

### Q1 how's your energy

* :+1: i feel energetic from this week
* :-1: i feel exhausted from this week

## note taking

please use this discussion as a place to take notes. create a comment that you edit throughout the lesson. 

_the perfect is the enemy of the good_

## agenda

0. [:00-:07] chit chat
1. show and tell
2. read me, read you
3. you and your media
4. homeworks
5. [:50-:60] wrapup

write! right?

**Comment by isabela-pf** [comment]
- Kush's blog! https://krshrimali.github.io/
- Mars' update on drafted NumPy blog post and illustrations
- Matthew shared [highlight-twitter-alt-text](https://github.com/alexwlchan/highlight-twitter-alt-text)
- "it's okay to hate yourself as long as you hate your past self…get better every day." —tony
- A&R = agents and representation?
- "at least it's not bitcoin"

"How do you know you're getting better at things?"
- Different things matter to you at different times. Tracking those metrics when they matter can help.
- Changing the scale you are looking at. Look at work from years ago and it will probably be easier to compare how you've changed than work from a month ago.
- Sometimes keeping drafts can help because you can track the process.
- Sometimes it's feedback from other people.
